packages/jest-reporters/package.json

```diff
@@ -3,15 +3,22 @@
   "description": "Jest's reporters",
   "version": "24.1.0",
   "main": "build/index.js",
+  "types": "build/index.d.ts",
   "dependencies": {
+    "@jest/core": "^24.1.0",
```

Comment:
Cannot depend on core, that creates a cyclical dependency. If you need some type, add it to jest-types